README.md

```
@@ -26,16 +26,18 @@ This custom integration tracks devices connected to your AT&T router by scraping
 
 ### Initial Setup
 
-#### Option 1: Using Device Access Code (Recommended - Coming Soon)
-*Note: Direct authentication with Device Access Code is planned for a future update. For now, please use Option 2.*
+#### Option 1: Using Device Access Code (Recommended)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)